lapce-data/src/palette.rs

```diff
@@ -761,12 +761,20 @@ impl PaletteViewData {
 
     #[allow(unused_variables)]
     fn get_commands(&mut self, ctx: &mut EventCtx) {
+        let excluded_items: Vec<String> = vec![
```

Comment:
Could this be a `const` slice of `&'static str` instead? I don't think it's necessary to run through the allocator twice here.